src/Esfa.Vacancy.Application/Interfaces/ITrainingDetailService.cs

```diff
@@ -7,5 +7,7 @@ public interface ITrainingDetailService
     {
         Task<Framework> GetFrameworkDetailsAsync(int code);
         Task<Standard> GetStandardDetailsAsync(int code);
+        Task<TrainingDetail> GetFrameworkDetailsAsync(string frameworkCode);
```

Comment:
Just a suggestion, maybe call this frameworkLarsCode knowing that it is the public lars identifier